indico/modules/events/tracks/templates/management.html

```diff
@@ -1,20 +1,29 @@
-{% extends 'layout/full_content_base.html' %}
+{% extends 'layout/base.html' %}
+{% from 'events/tracks/_track_list.html' import render_track_list %}
 
 {% block title %}
     {%- trans %}Programme{% endtrans -%}
 {% endblock %}
 
-{% block content %}
-    {% for track in tracks %}
-        <div>
-            {{ track.title }}
-        </div>
-    {% endfor %}
+{% block description %}
+    {%- trans -%}
+        Here you can find a list of tracks for this event.
+        You can add, edit, remove or sort these tracks.
+    {%- endtrans -%}
+{% endblock %}
 
+{% block content %}
     <a class="i-button highlight icon-plus"
         data-href="{{ url_for('.create', event) }}"
+        data-title="{% trans %}Add new track{% endtrans %}"
         data-ajax-dialog
         data-reload-after>
```

Comment:
Why are we still reloading the page here instead of using `data-update` like for edit/delete?